gvsbuild/groups.py

```diff
@@ -31,6 +31,7 @@ def __init__(self):
             dependencies = [
                 'cmake',
                 'meson',
+                'msys',

```

Comment:
excuse me for my ignorance, but isn't msys2 (required by gvsbuild) supposed to be the new and improved version of msys?

Reply:
forget my comment sorry, of course you do mean msys2 here.
by the way, thanks again for working on this!

Reply:
It's msys2 but all over the script is named msys (e.g. def exec_msys(...), opts.msys_dir, ...) so I keep the same name.

Probably is correct and better call the project, in the tools, and in the dependencies, msys2.